Documentation/Publications.md

```diff
@@ -57,185 +57,186 @@ _If your work uses **MBDyn**, please cite these papers as appropriate_:
-* O. Verlinden, G. Kouroussis, S. Datoussaid, C. Conti, "**[Open Source Symbolic and Numerical Tools for the Simulation of Multibody Systems](http://mecara.fpms.ac.be/EasyDyn/paper.pdf)**" [Faculté Polytechnique de Mons](http://mecara.fpms.ac.be/EasyDyn/) 6th National Congress on Theoretical and Applied Mechanics, 26-27 May 2003, Gent (Belgium). 
-* H. van der Ven and O.J. Boelens, "**Affordable CFD Simulations of Rotors in Forward Flight - A Feasibility Study with Future Application to Vibrational Analysis**" Netherlands National Aerospace Laboratory ([NLR](http://nlr.nl)) presented at the _59th Forum of the [American Helicopter Society](http://vtol.org)_, May 6-8 2003, Phoenix, Arizona. 
-* "**[Cooperative Research and Development Agreement (CRADA) for Multibody Formulation for Tiltrotor Aeroelastic Analysis](https://www.aero.polimi.it/~ghiro/crada.htm)**" between Vehicle Technology Center (VTC) U.S. Army Research Laboratory (ARL) and [Politenico di Milano](http://polimi.it) (DAE/POLIMI) [Dipartimento di Ingegneria Aerospaziale](http://www.aero.polimi.it). 
-* **[GVT Test Plan](http://sonicboom.larc.nasa.gov/wrats/gvtplan.html)** for the WRATS Soft-Inplane Rotor System in the _Wing and Rotor Aeroelastic Testing System_ page at [NASA](http://www.nasa.gov) [LaRC](http://www.larc.nasa.gov)'s Aeroelasticity Branch (sometimes unreachable, sorry) 
-* William T. Yeager, Jr., and Raynond G. Kvaternik "**[A Historical Overview of Aeroelasticity Branch and Transonic Dynamics Tunnel Contributions to Rotorcraft Technology and Development](http://techreports.larc.nasa.gov/ltrs/PDF/2001/tm/NASA-2001-tm211054.pdf)**" NASA/TM-2001-211054, ARL-TR-2564, 2001. 
-* William T. Yeager Jr., Raymond G. Kvaternik "**[Contributions of the Langley Transonic Dynamics Tunnel to rotorcraft technology and development](http://www.cs.odu.edu/~mln/ltrs-pdfs/NASA-aiaa-2000-1771.pdf)**" AIAA-2000-1771 AIAA/ASME/ASCE/AHS/ASC Structures, Structural Dynamics, and Materials Conference and Exhibit, 41st, Atlanta, GA, Apr. 3-6, 2000. 
-* Thomas Hablowetz "**Advanced Helicopter Flight and Aeroelastic Simulation based on General Purpose Multibody Code**" presented at the _AIAA Modeling and Simulation Technologies Conference and Exhibit_ Denver, CO, USA, August 14-17, 2000 (AIAA 2000-4299) 
+## Conference Papers:
+* E. P. Okabe, J. H. Izuka, R. Resch ["**Analysis of Analytical Hydrodynamic Bearing Models on a Reciprocating Compressor**"](https://www.researchgate.net/publication/327108298_Analysis_of_Analytical_Hydrodynamic_Bearing_Models_on_a_Reciprocating_Compressor_Vol_1) 10th International Conference on Rotor Dynamics – [IFToMM 2018](https://link.springer.com/chapter/10.1007/978-3-319-99262-4_22), Rio de Janeiro, 23-27 September 2018
```

Comment:
If I get it right, this paper uses OpenModelica, and no mention is made about MBDyn. Why cite it here? Does the implementation/model matches your work in MBDyn (likely yes, but perhaps we can explain the citation)?

Reply:
Well, Eduardo Okabe used the old Fortran 90 code from module-hydrodynamic_plain_bearing for his paper. Actually, that's the only reason why I'm one of authors. In the meantime, the old Fortran code was converted to C++. Nevertheless, it is still the same algorithm.

Reply:
OK, I have added this remark and link:
![image](https://github.com/user-attachments/assets/4f1a123e-364a-4827-a0b2-648c1b9ca31c)

Reply:
Merged and published on www.mbdyn.org .
Thank you!